pact/Pact/Core/IR/Eval/CoreBuiltin.hs

```diff
@@ -1656,8 +1735,10 @@ coreDescribeNamespace :: (CEKEval step b i m, MonadEval b i m) => NativeFunction
 coreDescribeNamespace info b cont handler _env = \case
   [VString n] -> do
     pdb <- viewEvalEnv eePactDb
+    chargeGasArgs info $ GRead $ sizeOf SizeOfV0 n
```

Comment:
HMM. This does not seem right. Why are we charging gas on the size of the name of the namespace?

Reply:
Should charge for an object with the name + two guards (user and admin one)

Reply:
The namespace itself is only charged if it's found by name. Hence to prevent undercharging in case the namespace isn't found, it's easier and less error-prone to just charge for the mere fact of looking up the name in the DB. _After_ it's found, it's also charged for the namespace object with its guards.